node/node.go

```diff
@@ -356,43 +358,134 @@ func (node *Node) Start() error {
 	if len(allTopics) == 0 {
 		return errors.New("have no topics to listen to")
 	}
-	weighted := make([]*semaphore.Weighted, len(allTopics))
-	const maxMessageHandlers = 200
-	ctx := context.Background()
+
+	const (
+		maxMessageHandlers = 200
```

Comment:
do we have data that we've used all the cost of 200? shall we adjust the number 200 or not?

Reply:
Couldn’t deploy as much as needed over weekend , roughly speaking takes about 10 handlers at normal operating times